readme.txt

```diff
@@ -35,11 +35,13 @@ To install the External Videos plugin simply:
 
 = What sites are supported? =
 
-Currently supported sites are: YouTube, Vimeo, DotSub
+Currently supported sites are: YouTube, Vimeo
```

Comment:
Just making screenshots now, and one more option to "add external-videos to the main post loop" which i use at the kiteboat site, might be useful for others